forte/data/entry_type_generator.py

```diff
@@ -10,6 +11,15 @@ def _get_type_attributes():
     return type_attributes
 
 
+def _get_entry_attribute_by_class(input_entry_class: str):
```

Comment:
though docstring for private classes is not required, still helpful to have for clarity